scripts/tools/check_includes_config.py

```diff
@@ -188,7 +188,7 @@
     'src/controller/CHIPDeviceController.cpp': {'string'},
     'src/qrcodetool/setup_payload_commands.cpp': {'string'},
     'src/access/AccessRestrictionProvider.h': {'vector', 'map'},
-
```

Comment:
Is this to be used by the device side? If so, this should not be overridden without justification.

Reply:
I will remove it.